tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_torch.py

```diff
@@ -0,0 +1,979 @@
+import json
+import logging
+import os
+import random
+import time
+import warnings
+
+import torch
+import torch.nn as nn
+import numpy as np
+import pandas as pd
+
+from autogluon.core import Space
+from autogluon.core.constants import QUANTILE
+from autogluon.core.utils import try_import_torch
+from autogluon.core.models.abstract import model_trial
+from autogluon.core.utils.exceptions import TimeLimitExceeded
+from autogluon.core import args
+from autogluon.core.models.abstract.abstract_model import AbstractNeuralNetworkModel
+from .embednet import getEmbedSizes
+from .tabular_nn_model import TabularNeuralNetModel
+from ..utils import fixedvals_from_searchspaces
+
+
+logger = logging.getLogger(__name__)
+
+
+class TabularNeuralQuantileModel(TabularNeuralNetModel):
```

Comment:
I don't think moving TabularNeuralQuantileModel into tabular_nn_torch.py will work because it seems like it'll break the lazy import?  
We need to ensure `try_import_torch()` is called before any `import torch` ever executes (including upon `import autogluon.tabular`).  So TabularNeuralQuantileModel may need to be in its own file.